tests/ui_tools/test_popups.py

```diff
@@ -187,6 +188,131 @@ def test_feature_level_content(self, mocker, zulip_version):
         )
 
 
+class TestUserInfoView:
+    @pytest.fixture(autouse=True)
+    def mock_external_classes(self, mocker):
+        self.controller = mocker.Mock()
+        mocker.patch.object(self.controller, 'maximum_popup_dimensions',
+                            return_value=(64, 64))
+        mocker.patch(VIEWS + '.urwid.SimpleFocusListWalker', return_value=[])
+        self.user_data = dict(
```

Comment:
This is essentially a fixture for the output of `get_tidied_user_info`?

I understand for getting this going that a dict internal to the test is easier, but it would be good if we could unify a "sample output" from that method, ie. relate it to our current standard test users in some way. If we change the output from the model method then we need to change the internals of this test otherwise - or eg. handling adding is_moderator.